### ISSUE TITLE
[BUG]: App checker found Accessibility issues in Power Cat Component 

### ISSUE THREAD

**Issue by pramodrshinde1985** [microsoft/powercat-creator-kit#409]
**Describe the bug**

We have incorporated the Power CAT component into our application, but we are actively addressing accessibility concerns. Some of these concerns are related to controls within custom components that we are unable to resolve directly since they are third-party components. Therefore, we kindly request assistance in resolving the accessibility issues specifically pertaining to the 'Dialog' control.

**To Reproduce**
1. Go to environment "Procurement BI"
2. Go to Apps --> click on edit Outsource Compliance Tool 
3. Scroll down to 'App Checker.'
5. See error
<img width="823" alt="image" src="https://github.com/microsoft/powercat-creator-kit/assets/145493052/110ad2b5-bb4e-4263-a98a-b69f91ded40c">

**Expected behavior**
We are expecting all Accessibility issues are resolved including third party components.
